warehouse/accounts/views.py

```diff
@@ -133,10 +143,15 @@ def logout(request):
         #       to handle this for us.
         request.session.invalidate()
 
+        redirect_to = request.POST.get(redirect_field_name,
+                                       request.GET.get(redirect_field_name))
+
+        # Security check -- don't allow redirection to a different host.
+        if not is_safe_url(url=redirect_to, host=request.host):
+            redirect_to = request.path
```

Comment:
I know this is what Django does, but it kind of seems silly to redirect back to the logout view after they've successfully been logged out just because the ``redirect_to`` isn't valid. Probably this should just use the default view. That doesn't actually exist until #492 lands, so I'd just hardcode it to ``"/"`` for now.